tests/network/local_network.go

```diff
@@ -0,0 +1,34 @@
+package network
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"math/big"
+
+	"github.com/ava-labs/subnet-evm/core/types"
+	"github.com/ava-labs/teleporter/tests/utils"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// Implements Network, pointing to the network setup in local_network_setup.go
+type LocalNetwork struct{}
```

Comment:
can you add top of file check that `LocalNetwork` implements `Network` interface?